### PR TITLE
initial server created

### PR DESCRIPTION
**Pull request by imansedky5**
-  Install express package
-  Create app using express 
- Server listen to port 5000
- Added  Scripts to Package.json file (start,dev, kill)
- Added  Add Proxy to react folder package.josn
- installed  server packages (joi, boom, supertest, compression, cookie-parser, pg)
- made changes in package.json file(scripts)
#47

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const { app } = require('./app');
+
+const port = app.get('port');
+
+app.listen(port, () => console.log('hey from port 5000'));
```
Comment (hassanelnajjar) [inline server/index.js:5]: In app.js file, you set the port but you didn't use it here

File: package.json [hunk 2]
```diff
@@ -18,9 +21,11 @@
...
   "homepage": "https://github.com/GSG-G9/calorie-tracker#readme",
   "dependencies": {
+    "dotenv": "^8.2.0",
     "express": "^4.17.1"
   },
   "devDependencies": {
     "jest": "^26.6.0"
-  }
+  },
+  "proxy": "http://localhost:4000"
```
Comment (hassanelnajjar) [inline package.json:30]: please,change the port number, I set the port 5000 for server not 4000

**Review by hassanelnajjar — CHANGES_REQUESTED** [review]
branch name should begin with the issue number like 47-create-initial-server

**Review by hassanelnajjar — APPROVED** [review]

**Review by hassanelnajjar — APPROVED** [review]

**Review by khamis-hesham — APPROVED** [review]

**Review by zeinjendeya — DISMISSED** [review]

**Review by hassanelnajjar — DISMISSED** [review]

File: .gitignore [hunk 3]
```diff
@@ -1,2 +1,3 @@
 node_modules
-.env
\ No newline at end of file
+.env
+client/.eslintcache
```
Comment (lina-jamal) [inline .gitignore:3]: add new line at the end

File: client/.eslintcache [hunk 4]
```diff
@@ -0,0 +1 @@
+[{"/home/iman/calorie-tracker/client/src/index.js":"1","/home/iman/calorie-tracker/client/src/App.js":"2"},{"size":168,"mtime":1612179889152,"results":"3","hashOfConfig":"4"},{"size":148,"mtime":1612179889152,"results":"5","hashOfConfig":"4"},{"filePath":"6","messages":"7","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"8"},"ac7ule",{"filePath":"9","messages":"10","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"/home/iman/calorie-tracker/client/src/index.js",[],["11","12"],"/home/iman/calorie-tracker/client/src/App.js",[],{"ruleId":"13","replacedBy":"14"},{"ruleId":"15","replacedBy":"16"},"no-native-reassign",["17"],"no-negated-in-lhs",["18"],"no-global-assign","no-unsafe-negation"]
```
Comment (lina-jamal) [inline client/.eslintcache:1]: put this file in gitignore

**Review by zeinjendeya — APPROVED** [review]

**Review by lina-jamal — APPROVED** [review]

**Review by hassanelnajjar — APPROVED** [review]

**Review by khamis-hesham — APPROVED** [review]

**Pull request merged**